backends/hif/pk-backend-hif.c

```diff
@@ -186,7 +187,7 @@ pk_backend_setup_hif_context (HifContext *context, GKeyFile *conf, const gchar *
 	hif_context_set_cache_dir (context, cache_dir);
 	solv_dir = g_build_filename (destdir, "/var/cache/PackageKit", release_ver, "hawkey", NULL);
 	hif_context_set_solv_dir (context, solv_dir);
-	repo_dir = g_build_filename (destdir, "/etc/yum.repos.d", NULL);
+	repo_dir = g_build_filename (destdir, hif_get_reposdir(), NULL);
```

Comment:
I am not sure how DNF does fallback between /etc/yum.repos.d/ and /etc/distro.repos.d/, but I suspect it may be necessary to look at _both_ directories, instead of just one or the other, in order to support 3rd party repos that may drop .repo files in either of the directories.